examples/fully-featured/flake.nix

```diff
@@ -5,7 +5,8 @@
     nixpkgs.url = github:nixos/nixpkgs/release-20.09;
     unstable.url = github:nixos/nixpkgs/nixos-unstable;
     nur.url = github:nix-community/NUR;
-    utils.url = github:gytis-ivaskevicius/flake-utils-plus;
+    # utils.url = github:gytis-ivaskevicius/flake-utils-plus;
```

Comment:
this comment probably should be removed

Reply:
Thanks for the catvh! Thats actually not quite what I originally had. Thanks for spotting. The uncommented one should be a relative path two directoriss up. Will fix that.